src/status/myDomainResolver.ts

```diff
@@ -108,6 +108,17 @@ export class MyDomainResolver extends AsyncOptionalCreatable<MyDomainResolver.Op
     return ensureString(await client.subscribe());
   }
 
+  public async getCnames(): Promise<string[]> {
+    try {
+      await this.resolve();
```

Comment:
make sure host is reachable

Reply:
what would happen on unresolvable if this weren't here?  wouldn't the next line still catch and return `[]` ?